README.md

```diff
@@ -60,59 +63,74 @@ mamba install wisdem -y
 pip install -e .[dev,docs]
 ```
 
-To test the installation, from the `Ard` folder run unit and regression tests:
+## Testing instructions
+
+The installation can be tested comprehensively using pytest.
+The developers also provide some convenience scripts for testing new installations; from the `Ard` folder run unit and regression tests:
```

Comment:
The tests can be run with simply running `pytest` from the top directory. I'm inclined to suggest that approach for new users as it is more approachable.